app/src/main/java/com/infomaniak/mail/ui/main/thread/actions/JunkBottomSheetDialog.kt

```diff
@@ -65,7 +65,6 @@ class JunkBottomSheetDialog : ActionsBottomSheetDialog() {
             createDescriptionDialog(
                 title = getString(R.string.reportPhishingTitle),
                 description = getString(R.string.reportPhishingDescription),
-                confirmButtonText = R.string.buttonReport,
```

Comment:
Is this linked to the italian strings updates

Reply:
Yes because this string is deleted during strings update